tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -94,8 +98,14 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
     std::string byzantineStrategies;
     bool is_separate_communication_mode = false;
     int addAllKeysAsPublic = 0;
+    int stateTransferMsgDelayMs = 0;
 
+    // do not change order of the next options, as some might use an option index!
     static struct option longOptions[] = {
+        // long format only options should be put here
+        {"delay-state-transfer-messages-millisec", required_argument, 0, 0},
```

Comment:
I think this should be an optional argument, since the byzantine strategies are. Also since it is a long format parameter, maybe the `stateTransferMsgDelayMs` variable can be used with a flag, which will save all the parsing.